addons/mail/static/src/components/composer_suggested_partners/composer_suggested_partners.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-name="mail.ComposerSuggestedPartners" owl="1">
+        <!-- List of followers -->
+        <div class="o_composer_suggested_partners">
```

Comment:
convention for naming (structural) classes:
Root node:
```
o_<ComponentName>
```
(non-root) node of template:
```
o_<ComponentName>_<partName>
```

Modifier on a node of template:
```
o-<modifierName>
OR
o_<ComponentName>_<partName>-<modifierName>
```
(use 2nd format if there's already conflicting style with a global `o-<modifier>`, like `o-active`)